wallet/src/main/scala/org/bitcoins/wallet/Wallet.scala

```diff
@@ -30,6 +37,380 @@ sealed abstract class Wallet extends LockedWallet with UnlockedWalletApi {
     LockedWallet(nodeApi, chainQueryApi)
   }
 
+  /** We need a output so we can get proper tx size estimation */
```

Comment:
can we move all DLC specific stuff into it's own trait and then have it extended by the `Wallet`?